### ISSUE TITLE
reverb cutting out

### ISSUE THREAD

**Issue by cacogen** [HippieStationArchive/HippieStation13-2#445]
the reverb introduced by goonsound frequently cuts out mid-sound and then sometimes kicks back in

**Comment by JohnGinnane** [comment]
Yeah, I've noticed the same. Could happen while the sounds is playing, noticable when walking around
